pkg/daemon/fake/fake_manager.go

```diff
@@ -0,0 +1,117 @@
+//go:build windows
```

Comment:
To me it seems like the naming scheme for these files should be:
pkg/daemon/fake/fake_manager.go -> pkg/daemon/fake/manager.go

Reply:
Applies to all fake package files